js/viewer/annotation/path.js

```diff
@@ -5,10 +5,6 @@ function Path(attrs){
 
     this._attrs["tag"] = "path";
     this._attrs["d"] = attrs["d"] || "";
-    this._attrs["fill"] = attrs["fill"] || "";
```

Comment:
Why did you remove these? and why didn't you remove them from the other prototypes (other shapes)?

Reply:
I removed `fill` from path because it is already present in `base.js`. Fill is not there in `circle` and `rect`, and if you were talking about 'polyline` and `polygon` then, they are just empty container files for future use, so I did not think necessary to make changes in then. I can do so if you want to.

Reply:
Yes, that's what I'm talking about. Remove it from the other prototypes as well. It doesn't matter that they're not currently used, if you don't change the code consistently, we won't remember why we did it that way.

Reply:
Okay, so I will also remove other things that are redundant or not being used.